ar: no archive members specified

1.9.51 doesn't build for me on the Mac:
```
+ /Users/djs/.opam/system/bin/ocamlfind ocamlopt -a -o lib/rpclib.cmxa
ar: no archive members specified
usage:  ar -d [-TLsv] archive file ...
	ar -m [-TLsv] archive file ...
	ar -m [-abiTLsv] position archive file ...
	ar -p [-TLsv] archive [file ...]
	ar -q [-cTLsv] archive file ...
	ar -r [-cuTLsv] archive file ...
	ar -r [-abciuTLsv] position archive file ...
	ar -t [-TLsv] archive [file ...]
	ar -x [-ouTLsv] archive [file ...]
File "_none_", line 1:
Error: Error while creating the library lib/rpclib.a
Command exited with code 2.
Compilation unsuccessful after building 99 targets (0 cached) in 00:00:10.
E: Failure("Command ''/Users/djs/.opam/system/bin/ocamlbuild' lib/rpclib_core.cma lib/rpclib_core.cmxa lib/rpclib_core.a lib/rpclib_core.cmxs lib/xml.cma lib/xml.cmxa lib/xml.a lib/xml.cmxs lib/json.cma lib/json.cmxa lib/json.a lib/json.cmxs lib/cmdlinergen.cma lib/cmdlinergen.cmxa lib/cmdlinergen.a lib/cmdlinergen.cmxs lib/htmlgen.cma lib/htmlgen.cmxa lib/htmlgen.a lib/htmlgen.cmxs lib/markdowngen.cma lib/markdowngen.cmxa lib/markdowngen.a lib/markdowngen.cmxs lib/rpclib_lwt.cma lib/rpclib_lwt.cmxa lib/rpclib_lwt.a lib/rpclib_lwt.cmxs lib/rpclib_async.cma lib/rpclib_async.cmxa lib/rpclib_async.a lib/rpclib_async.cmxs lib/rpclib_syntax.cma lib/rpclib_syntax.cmxa lib/rpclib_syntax.a lib/rpclib_syntax.cmxs lib/rpclib_unix.cma lib/rpclib_unix.cmxa lib/rpclib_unix.a lib/rpclib_unix.cmxs lib/rpclib.cma lib/rpclib.cmxa lib/rpclib.a lib/rpclib.cmxs lib/idl.cma lib/idl.cmxa lib/idl.a lib/idl.cmxs ppx/ppx_deriving_rpc.cma ppx/ppx_deriving_rpc.cmxa ppx/ppx_deriving_rpc.a ppx/ppx_deriving_rpc.cmxs -use-ocamlfind -plugin-tags 'package(cppo_ocamlbuild)' -tag debug' terminated with error code 10")
make: *** [build] Error 1
```

The same steps work on Linux:
```
/home/opam/.opam/4.04.1/bin/ocamlfind ocamlc -a -o lib/rpclib.cma
/home/opam/.opam/4.04.1/bin/ocamlfind ocamlopt -a -o lib/rpclib.cmxa
/home/opam/.opam/4.04.1/bin/ocamlfind ocamlopt -shared -I lib lib/rpclib.cmxa -o lib/rpclib.cmxs
```

It looks like the clang toolchain refuses to create empty archives. I think a single dummy module would fix it.

--- a/_oasis
+++ b/_oasis
@@ -95,6 +95,7 @@ Library rpclib_unix
 Library rpclib
   Path: lib
   FindlibName: rpclib
+  Modules: Rpc_empty_module
   BuildDepends: rpclib.xml
 
 Library idl

--- a/lib/rpc_empty_module.ml
+++ b/lib/rpc_empty_module.ml
@@ -1,0 +1,3 @@
+(* THe clang toolchain doesn't like archives with no elements,
+ * see https://github.com/mirage/ocaml-rpc/issues/44
+ *)

